database/database/connection.py

```diff
@@ -197,16 +197,30 @@ def __init__(self) -> None:
         from core.sustainability_labels.bootstrap_database import sustainability_labels
 
         with self._session_factory() as db_session:
-            # NOTE: this is slowly..
+            # NOTE: this is slowly...
             # if we have many more labels to bootstrap, we should refactor it.
+
             for label in sustainability_labels:
-                if (  # If label does not exist
+                if (  # If label exist with different timestamp, row will be updated
+                    not db_session.query(
+                        SustainabilityLabelsTable.id, SustainabilityLabelsTable.timestamp
+                    )
+                    .filter(
+                        SustainabilityLabelsTable.id == label.id,
+                        SustainabilityLabelsTable.timestamp == label.timestamp,
+                    )
+                    .first()
+                ):
+                    db_session.query(SustainabilityLabelsTable).filter(
+                        SustainabilityLabelsTable.id == label.id
+                    ).update(label.dict())
```

Comment:
I would prefer it if we do not update existing rows. Creating new ones allows us to version the labels' evaluations.

Reply:
File changed in [5367245](https://github.com/calgo-lab/green-db/pull/73/commits/536724540c938edbfbc7ba78342ac315edfe1813)